x-pack/plugins/apm/server/lib/rum_client/get_page_view_trends.ts

```diff
@@ -50,7 +40,17 @@ export async function getPageViewTrends({
             field: '@timestamp',
             buckets: 50,
           },
-          aggs: breakdownAggs,
+          aggs: breakdownItem
+            ? {
+                breakdown: {
+                  terms: {
+                    field: breakdownItem.fieldName,
+                    size: 9,
```

Comment:
Why `9`?

Reply:
This was mentioned in the issue that we need to show top 9 by volume https://github.com/elastic/kibana/issues/69712